### PR TITLE
fix: use bin/konvoy-image-wrapper now to fix podman tests 

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by dkoshkin — APPROVED** [review]

**Comment by dkoshkin** [comment]
Do we need the additional changes from https://github.com/mesosphere/konvoy-image-builder/pull/922/files?

File: .github/workflows/podman-aws-e2e.yaml [hunk 1]
```diff
@@ -40,22 +40,24 @@ jobs:
...
       - name: Build snapshot
-        run: make build.snapshot
+        run:  |-
+          make build-for-podman
+          goreleaser build --id konvoy-image-wrapper-for-podman
 
       - name: Run E2E test for AWS centos 7.9 using podman
         run: |-
-          KIB_CONTAINER_ENGINE=podman dist/konvoy-image-wrapper_linux_amd64_v1/konvoy-image build aws images/ami/centos-79.yaml --dry-run
+          KIB_CONTAINER_ENGINE=podman dist/konvoy-image-wrapper-for-podman_linux_amd64_v1/konvoy-image build aws images/ami/centos-79.yaml --dry-run
```
Comment (dkoshkin) [inline .github/workflows/podman-aws-e2e.yaml:60]: Why do we need a separate binary?

Comment (faiq) [reply .github/workflows/podman-aws-e2e.yaml:60]: build.snapshot is seeing disk space issues. i created a separate gorelaser target which only builds the amd64 image for podman

Comment (faiq) [reply .github/workflows/podman-aws-e2e.yaml:60]: furthermore, this allows us to execute the test using an embeded binary.

**Review by supershal — APPROVED** [review]
lgtm

**Pull request merged**